assets/scripts/main.js

```diff
@@ -1,40 +1,41 @@
-let books = JSON.parse(localStorage.getItem('booksData')) || [];
+class Collection {
+  constructor() {
+    this.arr = JSON.parse(localStorage.getItem('booksData')) || [];
+  }
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+  addBook(title, author) {
+    this.arr.push({ title, author });
+  }
 
-function addBook(title, author) {
-  const newBook = new Book(title, author);
-  books.push(newBook);
-}
+  removeBook(title, author) {
+    this.arr = this.arr.filter((elem) => elem.title !== title && elem.author !== author);
+  }
```

Comment:
Removing a book also remove all other books with the same title and/or author. :-1:  Kindly consider adding a unique `id` to each element to fix this issue and make sure only the selected book is removed. :pray: